pytorch_lightning/loggers/tensorboard.py

```diff
@@ -198,7 +198,7 @@ def log_graph(self, model: LightningModule, input_array=None):
                 input_array = model.example_input_array
 
             if input_array is not None:
-                input_array = model.transfer_batch_to_device(input_array, model.device)
+                input_array = model._prepare_batch_for_transfer(input_array)
```

Comment:
Isn't this missing `device=self.device`?

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/blob/a42d2d7ba44e2abef12804e32dba3c92de45d91e/pytorch_lightning/core/hooks.py#L559

the default will take `self.device`.